bin/kalite

```diff
@@ -78,6 +85,28 @@ if not os.path.exists(kalite_path):
 # Converts to normal string for Windows users, where it gets turned into a unicode string and errors out in subprocess
 env['KALITE_DIR'] = str(kalite_path)
 
+# TODO(benjaoming): Delegate responsibility to the .deb package
+if os.path.isfile("/etc/ka-lite/username"):
```

Comment:
Still, you should probably use `os.sep` in path names, and use [os.name](https://docs.python.org/2/library/os.html#os.name) to make OS-conditional behavior.